di_website/templates/blog/footnotes.html

```diff
@@ -1,10 +1,10 @@
-{% load string_utils wagtailcore_tags %}
+{% load string_utils wagtailcore_tags markdownify %}
 {% if page.footnotes_list %}
     <div class="m-notes">
       <h3 class="m-notes__title">Notes</h3>
       <ul class="m-notes__list">
         {% for item in page.footnotes_list %}
-          <li><sup><a id="{{ item.footnote_id }}" href="#{{ item.source_id }}">{{ forloop.counter }}</a></sup>{{ item.text | urlize }}</li>
+          <li><sup><a id="{{ item.footnote_id }}" href="#{{ item.source_id }}">{{ forloop.counter }}</a></sup>{{ item.text | urlize | markdownify }}</li>
```

Comment:
If I remember correctly, markdownify on its own should be able to urlize as well.

Reply:
Just gave it a test, you're correct. Pushed a commit with just markdownify.